fe/src/main/java/org/apache/doris/rewrite/FoldConstantsRule.java

```diff
@@ -58,14 +64,14 @@
 public class FoldConstantsRule implements ExprRewriteRule {
     public static ExprRewriteRule INSTANCE = new FoldConstantsRule();
 
-    private Multimap<String, FEFunctionInvoker> functions = ArrayListMultimap.create();
+    private ImmutableMultimap<String, FEFunctionInvoker> functions;
+    private ImmutableSet<String> nonNullResultWithNullParamFunctions;
```

Comment:
Need more comment here. I'm not very clear about this attribute's purpose

Reply:
ok, i will add some comments.